pkg/server/server_podman_test.go

```diff
@@ -40,7 +40,7 @@ func setUpServerAndController() (*Server, *PodController, error) {
 		panic("Error creating temporary directory")
 	}
 	defer os.RemoveAll(tmpdir)
-	podctl, err := NewPodController(tmpdir, true)
+	podctl, err := NewPodController(tmpdir, "podman")
```

Comment:
Either use the constant defined in runtime.go or remove the constant.

Reply:
good point, fixed